src/plugins/decor/ui/modals/GuidelinesModal.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Vencord, a Discord client mod
+ * Copyright (c) 2023 Vendicated and contributors
+ * SPDX-License-Identifier: GPL-3.0-or-later
+ */
+
+import { Link } from "@components/Link";
+import { ModalCloseButton, ModalContent, ModalFooter, ModalHeader, ModalRoot, ModalSize, openModal } from "@utils/modal";
+import { findByPropsLazy } from "@webpack";
+import { Button, Text } from "@webpack/common";
+
+import { settings } from "../../settings";
+import { cl, requireAvatarDecorationModal } from "../";
+import { openCreateDecorationModal } from "./CreateDecorationModal";
+
+const DecorationModalStyles = findByPropsLazy("modalFooterShopButton");
+
+export default function GuidelinesModal(props) {
```

Comment:
this export seems unused
```suggestion
function GuidelinesModal(props) {
```